bundles/org.openhab.ui/web/src/components/widgets/standard/list/oh-list-item.vue

```diff
@@ -15,16 +17,93 @@
-      <!-- <oh-placeholder-widget v-else-if="context.editmode" class="oh-column-item placeholder" @click="context.editmode.addWidget(context.component, null, context.parent, 'accordion')" /> -->
+      <generic-widget-component v-if="isRegularAccordion" :context="childContext(context.component.slots.accordion[0])" v-on="$listeners" />
+    </f7-accordion-content>
+    <f7-accordion-content slot="root" v-if="isEquipmentAccordion && !context.editmode">
+      <generic-widget-component :context="childContext(context.component.slots.accordion[0])" v-on="$listeners" />
     </f7-accordion-content>
     <oh-icon slot="media" v-if="config.icon" :icon="config.icon" height="32" width="32" :color="config.iconColor" :state="(config.item && config.iconUseState) ? context.store[config.item].state : null" />
     <span slot="media" v-else-if="config.fallbackIconToInitial && config.title && context.parent.component.config && context.parent.component.config.mediaList" class="item-initial">{{ config.title[0].toUpperCase() }}</span>
   </f7-list-item>
 </template>
 
 <style lang="stylus">
+.oh-accordion-item
+  .item-link .item-inner:after
+    transition-duration: 300ms
+
+  .list,
+  .block
+    margin-top: 0
+    margin-bottom: 0
+
+  .block
+    > h1,
+    > h2,
+    > h3,
+    > h4,
+    > p
+      &:first-child
+        margin-top: 10px
+
+      &:last-child
+        margin-bottom: 10px
+
+.list li.oh-accordion-item ul
+  ltr(
+    padding-left: 0
+
+)
+  rtl(
+    padding-right: 0
+
+)
+
+.list ul ul
+  padding-left: var(--f7-list-item-padding-horizontal)!important
+
+.media-list .oh-accordion-item .item-inner {
+    padding-right: calc(16px + 0px)
+    padding-right: calc(var(--f7-list-item-padding-horizontal) + var(--f7-safe-area-right))
+}
+
+.media-list .oh-accordion-item .item-title-row {
+    padding-right: @css{max(20px, calc(var(--f7-list-chevron-icon-area)))}
+}
+
+.media-list .oh-accordion-item > .item-content > .item-inner > .item-title-row:before
+    font-family: 'framework7-core-icons'
+    font-weight: normal
+    font-style: normal
+    font-size: var(--f7-list-chevron-icon-font-size)
+    line-height: 1
+    letter-spacing: normal
+    text-transform: none
+    white-space: nowrap
+    word-wrap: normal
+    direction: ltr
+    -webkit-font-smoothing: antialiased
+    text-rendering: optimizeLegibility
+    -moz-osx-font-smoothing: grayscale
+    -moz-font-feature-settings: "liga"
+    font-feature-settings: "liga"
+    content: var(--f7-accordion-chevron-icon-down)
+    width: 14px
+    height: 8px
+    margin-top: -4px
+    line-height: 8px
+    right: 0
+    text-align: right
+    display: block
+    width: 100%
+    height: 100%
+    position: absolute
+    top: 50%
+    color: var(--f7-list-chevron-icon-color)
+
+.media-list .accordion-item-opened  > .item-content > .item-inner > .item-title-row:before
+    content: var(--f7-accordion-chevron-icon-up)
+
```

Comment:
I think these styles should be cleaned up:
- Stylus allows the syntax `property value` instead of `property: value` and it's widely adopted elsewhere so please change accordingly `margin-top: 10px` ➡️ `margin-top 10px`
- `oh-accordion-item` should probably be renamed to something more specific like `oh-equipment-accordion`
- Everything related to equipment accordions should be put beneath a `.oh-equipment-accordion` selector. Generic selectors like `.list ul ul` have the potential of breaking other things since the styles are not scoped to this component.